src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,30 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Arrays;
+import java.util.List;
+
 public class FileWork {
+
```

Comment:
this empty line is redundant

Reply:
further I will follow this comment everytime, thx